sdk/storage/Azure.Storage.Blobs/src/StorageRequestFailedException.cs

```diff
@@ -47,8 +47,13 @@ internal partial class StorageError
         /// <param name="clientDiagnostics">The <see cref="ClientDiagnostics"/> instance to use.</param>
         /// <param name="response">The failed response.</param>
         /// <returns>A RequestFailedException.</returns>
-        public Exception CreateException(ClientDiagnostics clientDiagnostics, Azure.Response response)
-            => clientDiagnostics.CreateRequestFailedExceptionWithContent(response, message: Message, content: null,  response.GetErrorCode(Code));
+        public Exception CreateException(ClientDiagnostics clientDiagnostics, Response response)
+            => clientDiagnostics.CreateRequestFailedExceptionWithContent(
+                response,
+                message: Message,
+                content: null,
+                response.GetErrorCode(Code),
+                AdditionalInformation);
```

Comment:
Can `DataLakeStorageError` at the bottom of the file have additional XML elements like `StorageError` does?  Should we copy the XML parsing code above here down into it too?

Reply:
`StorageError.CustomizeFromXml(...)` is called by the generated code.  I don't think adding this method to `DataLakeStorageError` will do anything, since the generated code won't call the method.

In addition, I looked into adding `AdditionalInformation` to `DataLakeStorageError`, but it appears this information is actually not provided by the DFS endpoint.